ghost/core/core/server/web/shared/middleware/mentions-limiter.js

```diff
@@ -0,0 +1,13 @@
+const url = require('url');
+const spamPrevention = require('./api/spam-prevention');
+
+module.exports = {
+    globalLimits(req, res, next) {
+        return spamPrevention.mentionsBlock().getMiddleware({
+            ignoreIP: false,
+            key(_req, _res, _next) {
+                _next(url.parse(_req.url).pathname);
```

Comment:
This should be the unique key for blocks - in this case I think we just want it as a static value